src/entities/party/party.service.ts

```diff
@@ -0,0 +1,20 @@
+import { Injectable } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
+import { Party } from './party.entity';
+
+@Injectable()
+export class PartyService {
+  constructor(
+    @InjectRepository(Party)
+    private readonly partyRepository: Repository<Party>,
+  ) {}
+
+  findAll(): Promise<Party[]> {
+    return this.partyRepository.find();
+  }
+
+  findOne(id: number): Promise<Party> {
+    return this.partyRepository.findOneBy({ id });
```

Comment:
Let's use everywhere the standard way to search by id. I mean we use 2 versions `findOne(id)`, `findOneBy({ id })` let's use one of them.

Reply:
Done. I've selected second version (`findOneBy({id}`)